src/utils/url.ts

```diff
@@ -28,9 +28,19 @@ export const venueRoomUrl = (venue: WithId<AnyVenue>, roomTitle: string) => {
   return venueRoom ? venueRoom.url : venueInsideUrl(venue.id);
 };
 
-export const isExternalUrl = (url: string) =>
-  url.includes("http") &&
-  window.location.host !== new URL(getRoomUrl(url)).host;
+export const isExternalUrl = (url: string) => {
+  try {
+    const urlHost = new URL(url).host;
+    const currentHost = window.location.host;
+    return currentHost !== urlHost;
```

Comment:
[NB] you still could have left this as a single line if you wanted, I think it's simple enough that breaking it out doesn't add much extra clarity.
```suggestion
    return new URL(url).host !== window.location.host;
```